label/label.go

```diff
@@ -90,6 +89,35 @@ func (m *GormLabelRepository) Create(ctx context.Context, u *Label) error {
 	return nil
 }
 
+// Save update the given label
+func (m *GormLabelRepository) Save(ctx context.Context, l Label) (*Label, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "label", "update"}, time.Now())
```

Comment:
Last value in array : `string{"goa", "db", "label", "save"}`

Reply:
Fixed in e6fd97cab919ee1db91e21c7f2207ac0802a4acb